### PR TITLE
Non-premium contraband price fix

### PR DESCRIPTION
**Pull request by BettonnCZ**
haha

## About The Pull Request

the contraband showed previously a different price than it actually charged, it should now be fixed

## Why It's Good For The Game

It won't confuse players wanting to buy said contraband

## Changelog
:cl:
fix: Non-premium contraband correctly charges the listed price
/:cl:

### CODE REVIEW

**Comment by St0rmC4st3r** [comment]
I think we should rather fix the broken prices for contraband.

**Comment by BettonnCZ** [comment]
well, this just works ¯\_(ツ)_/¯, i dont know where is the contraband  price referenced.

**Comment by Crossedfall** [comment]
This is the code you actually want to change. This checks to see if the item is hidden (contraband) and then applies the premium price to it. Just remove the `|| hidden_records.Find(R)` portion of the check.

https://github.com/BeeStation/BeeStation-Hornet/blob/a54b6c5682d2719b8ae6111785935d98e5be5cd9/code/modules/vending/_vending.dm#L766-L767

**Review by Crossedfall — CHANGES_REQUESTED** [review]
see above

**Comment by BettonnCZ** [comment]
obrazek
removed the  || hidden_records.Find(R)
obrazek
still shows the wrong price

**Comment by Crossedfall** [comment]
> 
> 
> obrazek
> removed the || hidden_records.Find(R)
> obrazek
> still shows the wrong price

Did you try to buy it though? It should only take the 10 credits from you.

**Comment by BettonnCZ** [comment]
i think it should be now good, i tought the vape is supposed to charge you 50, now it charges you 10 credits, which it displays.

File: code/modules/vending/cigarette.dm [hunk 1]
```diff
@@ -13,7 +13,7 @@
 					/obj/item/storage/box/matches = 10,
 					/obj/item/lighter/greyscale = 4,
 					/obj/item/storage/fancy/rollingpapers = 5)
-	contraband = list(/obj/item/clothing/mask/vape = 5)
+	contraband = list(/obj/item/clothing/mask/vape = 5)
```
Comment (Crossedfall) [inline code/modules/vending/cigarette.dm]:
```suggestion
	contraband = list(/obj/item/clothing/mask/vape = 5)
```

Buncha whitespace

**Comment by Crossedfall** [comment]
Fix the whitespace issue and then update your PR body to address the fact that you're fixing non-premium contraband pricing as a whole, not just vapes and I think we'll be good.

**Review by Crossedfall — APPROVED** [review]

**Pull request merged**